src/Components/Facility/CentralNursingStation.tsx

```diff
@@ -152,15 +152,12 @@ export default function CentralNursingStation({ facilityId }: Props) {
                       name="bed_is_occupied"
                       label="Hide Monitors without Patient"
                       value={
-                        qParams.bed_is_occupied === "true" ||
                         qParams.bed_is_occupied === undefined
+                          ? true
+                          : qParams.bed_is_occupied === "true"
                       }
                       onChange={({ name, value }) => {
-                        if (value) {
-                          updateQuery({ [name]: value });
-                        } else {
-                          removeFilter(name);
-                        }
+                        updateQuery({ [name]: value });
```

Comment:
are you sure this is right? we don't want to filter by `bed_is_occupied==False` when unchecked, rather we don't want to perform the filter itself based on `bed_is_occupied` when unchecked right?